src/Command/VersionCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Drupal\lightning\Command;
+
+use Drupal\Console\Core\Command\Command;
+use Drupal\Console\Core\Style\DrupalStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Yaml\Yaml;
+use Drupal\Console\Annotations\DrupalCommand;
+
+/**
+ * Class VersionCommand.
+ *
+ * @DrupalCommand
+ */
+class VersionCommand extends Command {
+
+  /**
+   * The Drupal application root.
+   *
+   * @var string
+   */
+  protected $appRoot;
+
+  /**
+   * VersionCommand constructor.
+   *
+   * @param string $app_root
+   *   The Drupal application root.
+   */
+  public function __construct($app_root) {
+    parent::__construct('lightning:version');
+    $this->appRoot = $app_root;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function configure() {
+    $this->setName('lightning:version');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    $finder = (new Finder())
+      ->files()
+      ->name('lightning.info.yml')
+      ->in($this->appRoot . '/profiles');
+
+    foreach ($finder as $info_file) {
+      $info = Yaml::parse($info_file->getContents());
+      if (($info['distribution']['name'] == 'Lightning') && isset($info['version'])) {
```

Comment:
Nit: There's an extra set of parentheses here. Also, let's use === when comparing strings; sometimes PHP does weird shit.